draft-ietf-quic-http.md

```diff
@@ -825,6 +821,70 @@ MUST be treated as a connection error of type HTTP_MALFORMED_FRAME.
 A server MUST treat a MAX_PUSH_ID frame payload that does not contain a single
 variable-length integer as a connection error of type HTTP_MALFORMED_FRAME.
 
+### INITIAL_PRIORITY {#frame-initial-priority}
+
+The INITIAL_PRIORITY (type=0x0F) frame specifies the client-advised priority of
```

Comment:
To maintain consistency with PRIORITY frame, you could use "sender-advised priority". Language elsewhere qualifies that the only sender that is allowed is a client.

or,

change it to client-advised in PRIORITY